src/components/side-panel/panel-toggle.js

```diff
@@ -46,13 +46,14 @@ const PanelTab = styled.div.attrs({
   align-items: flex-end;
   border-bottom-style: solid;
   border-bottom-width: 2px;
-  border-bottom-color: ${props => (props.active ? props.theme.subtextColorActive : 'transparent')};
+  border-bottom-color: ${props =>
+    props.active ? props.theme.subtextBorderColorActive : 'transparent'};
```

Comment:
this should be named `panelToggleBorderColor`

Reply:
Done